integrations/jenkins/Jenkinsfile

```diff
@@ -334,25 +340,31 @@ pipeline {
                     rm -fr $ORT_DATA_DIR/config
                     /opt/ort/bin/ort $ORT_OPTIONS download --project-url $ORT_CONFIG_VCS_URL $VCS_REVISION_OPTION -o $ORT_DATA_DIR/config
 
-                    if [ -f "$ORT_DATA_DIR/config/config.yml" ]; then
-                        echo "Not creating '$ORT_DATA_DIR/config/config.yml' as it already exists."
-                    else
-                        echo "Creating '$ORT_DATA_DIR/config/config.yml' template for job parameters."
-                        cat >$ORT_DATA_DIR/config/config.yml <<EOF
-                    ort:
-                      packageCurationProviders:
-                        - name: DefaultFile
-                        - name: DefaultDir
-                        - name: OrtConfig
-                          enabled: ${USE_ORT_CONFIG_CURATIONS}
-                        - name: ClearlyDefined
-                          enabled: ${USE_CLEARLY_DEFINED_CURATIONS}
-                    EOF
-                    fi
-
                     rm -f $HOME/.netrc
                     '''.stripIndent().trim()
                 }
+
+                script {
+                    env.ORT_CONFIG_DIR = "${env.ORT_DATA_DIR}/config/${env.ORT_CONFIG_VCS_PATH}"
```

Comment:
Should a trailing slash be stripped to avoid double slashes in the interpolated strings below, e.g. line 253?

Reply:
I've verified that this works as expected also if `ORT_CONFIG_VCS_PATH` is empty, so I'd prefer to omit that logic for simplicity.